Added markdown for ToLower() string method (#1845)
Codecademy/docs: content/c-sharp/concepts/strings/terms/tolower/tolower.md

---
Title: '.ToLower()'
Description: 'The ToLower() method is used to convert all the characters in a given string to lowercase.'
Subjects:
  - 'Computer Science'
  - 'Code Foundations'
Tags:
  - 'Strings'
  - 'Methods'
CatalogContent:
  - 'learn-c-sharp'
  - 'paths/computer-science'
---

The **`ToLower()`** method is a string method in C# that returns a new string in which all the characters are converted to lowercase. The method does not modify the original string, but rather returns a new one. This method is helpful in normalizing user input or comparing strings in a case-insensitive manner.
 

> 

Author: ok

## Syntax

```pseudo
string lowerCaseString = originalString.ToLower();
```

> 

Reviewer: ```suggestion
```
remove extra line break

Author: Okay

> Here, `originalString` is the string you want to convert to lowercase, and `lowerCaseString` is the new string that contains the lowercase characters.

Reviewer: ```suggestion
Here, `originalString` is the string being converted to lowercase, and `lowerCaseString` is the new string that contains the lowercase characters.
```

Author: do i need to remove the and as well ?

Author: okay

## Example

> In the following example, we create a new string `originalString` that contains the text *"HELLO WORLD"*. We then call the `ToLower()` method on this string and store the result in a new string variable `lowerCaseString`. Finally, we print the value of lowerCaseString to the console, which outputs *"hello world"*.

Reviewer: ```suggestion
In the following example, a new string `originalString` is created that contains the text "HELLO WORLD". Then the `.ToLower()` method is called on this string and store the result in a new string variable `lowerCaseString`. Finally, the value of `lowerCaseString` is printed to the console, which outputs "hello world".
```

Author: Okay will do

```cs
string originalString = "HELLO WORLD";
string lowerCaseString = originalString.ToLower();

Console.WriteLine(lowerCaseString); // Output: "hello world"
```

## Codebyte Example

> In following runnable example, we create a new string `originalString` that contains the text *"THIS IS A TEST STRING"*. We then call the `ToLower()` method on this string and store the result in a new string variable `lowerCaseString`. Finally, we print both the original and the lowercase strings to the console.

Reviewer: ```suggestion
In following runnable example, a new string `originalString` is created that contains the text "THIS IS A TEST STRING". Then the `.ToLower()` method is called on this string and the result is stored in a new string variable `lowerCaseString`. Finally, both the original and the lowercase strings are printed to the console.
```

Author: Got it, will change this one as well
